ic-http-outcall/README.md

```diff
@@ -0,0 +1,20 @@
+# HTTP Outcall Project
+=========================
+
```

Comment:
The entire documentation has to be revamped. 
It should contain:
- what is this project for
- what is the original problem that it solves
- how does it solve the problem
- why it is needed
- how to use it
- current limitations